modules/modMenu.js

```diff
@@ -147,7 +147,7 @@ const processes = {
       } else {
         const embed = u.embed()
         .setTimestamp()
-        .setAuthor(target.member.displayName + " 📌", target.member.user.displayAvatarURL())
+        .setAuthor({ name: target.member.displayName + " 📌", iconURL: target.member.user.displayAvatarURL() })
```

Comment:
This is one of the few places that did need updating. Keep the change you made here.